packages/botonic-react/src/webchat/menu.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext, useState } from 'react'
+import { WebchatContext } from '../contexts'
+import { Flex } from '@rebass/grid'
+import { Button } from '../components/button'
+import { WebchatMessageList } from './messageList'
+import Textarea from 'react-textarea-autosize'
+
+const DefaultMenu = () => {
+  const { webchatState } = useContext(WebchatContext)
+  return (
+    <Textarea
+      name="text"
+      minRows={2}
```

Comment:
Is this going to be always between 2 and 4 or in the future the user can define it?